server/command.go

```diff
@@ -104,9 +103,15 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 		case "settings":
 			handler = p.runSettingsCommand
 		default:
+			if command == "help" {
+				p.trackCommand(args.UserId, command)
+			} else {
+				p.trackCommand(args.UserId, "not found")
```

Comment:
Do we really want to track this? If yes, should we send the `command` as part of the telemetry data?

Reply:
I would consider the command part part of the "user data", since may be a mistake like `/todo Go for groceries`, and `Go` is part of the user input data.

I understand that it would be interesting in case there are misspelling or something similar, but I feel safer taking this approach.

Reply:
Good point, makes sense :+1: